backend/database/seeding/helpers.go

```diff
@@ -245,6 +254,38 @@ func newUserOperationPreferences(user models.User, op models.Operation, isFavori
 	}
 }
 
+func newUserGroupGen(first int64) func(name string, deleted bool) models.UserGroup {
+	id := iotaLike(first)
+	return func(name string, deleted bool) models.UserGroup {
+		if deleted {
```

Comment:
nice

Reply:
thanks! I can't take too much credit - most of this PR was just figuring out what other models (users, operations) were doing, and then taking it and modifying it slightly 😅